src/airmobility/object_a.proto

```diff
@@ -165,15 +167,16 @@ message VoxelValueWeather {
     CurrentWeather current_weather = 5 ; // 現況
 }
 
-message ObjectForecastWeather { // 気象予報オブジェクトの値。空間IDを含む。    
```

Comment:
There is no word "ForecastWeather" in English.
So I changed the name to "WeatherForecast".